wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -649,7 +639,17 @@ export class SDKv2Route {
     return 'test';
   }
 
-  tryFetchRedeemTx(txData: ParsedMessage): Promise<string | undefined> {
+  tryFetchRedeemTx(txData: TransferInfo): Promise<string | undefined> {
     throw new Error('Method not implemented.');
   }
+
+  async resumeIfManual(tx: TransactionId): Promise<routes.Receipt | null> {
+    const wh = await getWormholeContextV2();
+    const route = new this.rc(wh);
+    if (routes.isManual(route)) {
```

Comment:
As I'm adding the `nttRelay` route to this, I realized that it doesn't really make sense to have it extend `ManualRoute` since it would change the `AUTOMATIC_DEPOSIT` and `GAS_DROPOFF_SUPPORTED` properties on the route to `false`. I'm wondering if we could add a typeguard to the SDK, `isResumable` that checks if the `resume` function exists on the route object. All manual routes would be resumable (since they're required to implement it), and only some automatic routes are resumable, like nttRelay. What do you think?